simple-template/index.html

```diff
@@ -8,15 +8,17 @@
 <body>
   <h1>Simple template</h1>
 
-  <template id="my-paragraph">
+  <template id="custom-my-paragraph">
```

Comment:
After thinking about this, I came back to `custom-p-element` again (what I thought of first in the linked issue) what do you think?

```suggestion
  <template id="custom-p-element">
```